integration/sawtooth_integration/docker/xo-all-lang-smoke.yaml

```diff
@@ -0,0 +1,100 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+version: "2.1"
+
+services:
+
+  tp_config:
+    image: sawtooth-tp_config:$ISOLATION_ID
+    volumes:
+      - $SAWTOOTH_CORE:/project/sawtooth-core
+    expose:
+      - 40000
+    depends_on:
+      - validator
+    command: tp_config -vv tcp://validator:40000
+    stop_signal: SIGKILL
+
+  tp_xo_python:
+    image: sawtooth-tp_xo_python:$ISOLATION_ID
+    volumes:
+      - $SAWTOOTH_CORE:/project/sawtooth-core
+    expose:
+      - 40000
+    depends_on:
+      - validator
+    command: tp_xo_python -vv tcp://validator:40000
+    stop_signal: SIGKILL
+
+  tp_xo_javascript:
+    image: sawtooth-tp_xo_javascript:$ISOLATION_ID
+    volumes:
+      - $SAWTOOTH_CORE:/project/sawtooth-core
+    expose:
+      - 40000
+    depends_on:
+      - validator
+    command: tp_xo_javascript tcp://validator:40000
+    stop_signal: SIGKILL
+
+  validator:
+    image: sawtooth-validator:$ISOLATION_ID
+    volumes:
+      - $SAWTOOTH_CORE:/project/sawtooth-core
+    expose:
+      - 40000
+      - 8800
+    command: "bash -c \"\
+        sawtooth admin keygen && \
+        sawtooth admin genesis && \
+        validator --endpoint tcp://validator:8800 -v \
+            --bind component:tcp://eth0:40000 \
+            --bind network:tcp://eth0:8800 \
+    \""
+    stop_signal: SIGKILL
+
+  rest_api:
+    image: sawtooth-rest_api:$ISOLATION_ID
+    volumes:
+      - $SAWTOOTH_CORE:/project/sawtooth-core
+    expose:
+      - 40000
+      - 8080
+    depends_on:
+     - validator
+    command: rest_api -v --stream-url tcp://validator:40000 --host rest_api
```

Comment:
This will need to be updated to match the new rest api command line that was just merged.